TrafficCapture/dockerSolution/src/main/docker/migrationConsole/lib/console_link/console_link/models/cluster.py

```diff
@@ -105,7 +108,30 @@ def call_api(self, path, method: HttpMethod = HttpMethod.GET) -> requests.Respon
             f"{self.endpoint}{path}",
             verify=(not self.allow_insecure),
             auth=auth,
+            timeout=timeout
         )
         logger.debug(f"Cluster API call request: {r.request}")
         r.raise_for_status()
         return r
+
+    def execute_benchmark_workload(self, workload: str,
+                                   workload_params='target_throughput:0.5,bulk_size:10,bulk_indexing_clients:1,'
+                                                   'search_clients:1'):
+        client_options = ""
+        if not self.allow_insecure:
+            client_options += "use_ssl:true,verify_certs:false"
+        if self.auth_type == AuthMethod.BASIC_AUTH:
+            if self.auth_details['password'] is not None:
+                client_options += (f"basic_auth_user:{self.auth_details['username']},"
+                                   f"basic_auth_password:{self.auth_details['password']}")
+            else:
+                raise NotImplementedError(f"Auth type {self.auth_type} with AWS Secret ARN is not currently support "
+                                          f"for executing benchmark workloads")
+        elif self.auth_type == AuthMethod.SIGV4:
+            raise NotImplementedError(f"Auth type {self.auth_type} is not currently support for executing "
+                                      f"benchmark workloads")
+        logger.info(f"Running opensearch-benchmark with '{workload}' workload")
+        subprocess.run(f"opensearch-benchmark execute-test --distribution-version=1.0.0 "
```

Comment:
Do the benchmark outputs print to stdout or are they captured here? Kind of nice to show the user that something's happening, so I think I'm moderately in favor of not-capturing, but don't feel super strongly.